augur/tasks/start_tasks.py

```diff
@@ -88,7 +88,8 @@ def primary_repo_collect_phase(repo_git):
         #facade_phase(logger,repo_git),
         collect_linux_badge_info.si(repo_git),
         collect_releases.si(repo_git),
-        grab_comitters.si(repo_git)
+        grab_comitters.si(repo_git),
+        process_ossf_dependency_metrics.si(repo_git)
```

Comment:
This needs to stay in the secondary collection since it has a decorator of `@celery.task(base=AugurSecondaryRepoCollectionTask)
`